lightning/src/ln/channel.rs

```diff
@@ -1259,8 +1259,7 @@ impl<SP: Deref> Channel<SP> where
 				})
 			},
 			Channel::UnfundedInboundV1(chan) => {
-				let logger = WithChannelContext::from(logger, &chan.context, None);
-				let accept_channel = chan.signer_maybe_unblocked(&&logger);
+				let accept_channel = chan.signer_maybe_unblocked(logger);
```

Comment:
Why drop the logging context?

Reply:
Hmm... I see it used after the `unblock_chan` closure is called in `ChannelManager::signer_unblocked`, but not before.